assets/src/scss/editorOverrides.scss

```diff
@@ -280,3 +281,104 @@ input.describe[type=text][data-setting=caption] {
 .customize-control[id*="new_identity_styles"] label {
   display: inline;
 }
+
+// Media Archive Style Override
+.media-archive-content {
+  .archive-picker-title,
+  .nav-bulk-select,
+  .picker-sidebar-header .close-sidebar {
+    display: none;
+  }
+
+  .archive-picker {
+    gap: 0;
+  }
+
+  #import-to-library,
+  #import-to-library:disabled {
+    display: none;
+  }
+
+  .multiple-search {
+    margin-left: 20px;
```

Comment:
I have a small doubt with this.. don't we need to implement `RTL` here?

Reply:
What is RTL @GP-Dan-Tovbein ?

Reply:
I assume Right-to-Left. So maybe `margin-inline-start` could work better here.

Reply:
Yep! You can test it out by modifying the `direction` (`document.body.style.direction = "rtl"` or `ltr`)  through the browser inspector.